npm/private/npm_translate_lock_state.bzl

```diff
@@ -193,15 +180,10 @@ def _init_patches_labels(priv, rctx, label_store):
 
 ################################################################################
 def _init_patched_dependencies_labels(priv, rctx, label_store):
-    if rctx.attr.update_pnpm_lock:
-        # Read patches from package.json `pnpm.patchedDependencies`
```

Comment:
There is another instance of this divergence where `_load_root_package_json` is called in `_copy_unspecified_input_files`

Reply:
If we remove it from there then we can drop the entire `_load_root_package_json` block

Reply:
I'm wondering if we should actually do the opposite and always load `pnpm.*` from the package.json instead of lockfile.

Some things like the `pnpm.onlyBuiltDependencies` seem to not end up in the lockfile at all. Others like this exact one (`pnpm.patchedDependencies`) are in lockfile 5.4+ but not 5.3. But unless one of these `pnpm.*` options effects package versions there really is no reason for it to be required in a lockfile...

Reply:
I'm going to leave `_load_root_package_json` as-is since we will need it for other `pnpm.*` properties in a followup.